tools/deps/PackageLockWhitelistTest.java

```diff
@@ -42,7 +42,8 @@
   private List<String> packageLockLines;
   private List<Dependency> parsedDependencies;
 
-  private String MAVEN_CENTRAL_URL = "https://repo.maven.apache.org/maven2/";
+  @SuppressWarnings({"checkstyle:AbbreviationAsWordInName", "checkstyle:MemberName"})
+  private static String MAVEN_CENTRAL_URL = "https://repo.maven.apache.org/maven2/";
```

Comment:
Do we still need the suppression?

Reply:
@oferb we don't, fixed in https://github.com/google/startup-os/pull/490/commits/630402f6a6a747564fc9d1b341bb508efae0591a